### ISSUE TITLE
Fermenter hysteresis

### ISSUE THREAD

**Issue by MikonosII** [Manuel83/craftbeerpi3#68]
The fermenter hysteresis requires values in the heater component even if no heater actor exists in the fermenter hardware setup. With out some values in for the heater on / off the auto won't work.

**Comment by carlallen** [comment]
It works fine for me. Do you have values in all 4 offset fields? What does your log say?

**Comment by helibrewer** [comment]
Auto mode works but you will see errors for the function heater_off(), I assume because no heater actor is present
[app.txt](https://github.com/Manuel83/craftbeerpi3/files/1133834/app.txt)

**Comment by carlallen** [comment]
Yeah, it will show errors in the log, but shouldn't break anything. It's probably not a bad idea to verify that there is a heater configured before turning it on/off, but since that doesn't actually break anything it's pretty low priority in the fixes.

**Comment by berazafi** [comment]
I had the same problem.  Without any values entered into the heating offset fields, the Auto function would not turn on the cooling

**Comment by carlallen** [comment]
Yeah, even if you are not using heating, you still have to fill in all offset fields.

**Comment by Fuempel** [comment]
This can be more easily detected when using PR #221
